apps/finance/app/src/components/TransferRow.js

```diff
@@ -86,7 +90,11 @@ class TransferRow extends React.PureComponent {
           </time>
         </NoWrapCell>
         <NoWrapCell>
-          <IdentityBadge networkType={network.type} entity={entity} />
+          <LocalIdentityBadge
+            networkType={network.type}
+            entity={entity}
+            address={entity}
```

Comment:
Similar to https://github.com/aragon/aragon-apps/pull/722#discussion_r270042117.